controllers/addresspool_controller.go

```diff
@@ -72,23 +81,52 @@ func (r *AddressPoolReconciler) syncMetalLBAddressPool(instance *metallbv1alpha1
 	data.Data["Addresses"] = instance.Spec.Addresses
 	objs, err := render.RenderDir(AddressPoolManifestPath, &data)
 	if err != nil {
-		return errors.Wrapf(err, "Fail to render address-pool manifest")
+		return fmt.Errorf("Fail to render address-pool manifest %v", err)
```

Comment:
Just curious, why this change?

Reply:
I wanted to use different errors pkg to check for Notfound

Reply:
You can use named imports for that:
https://stackoverflow.com/questions/10408646/how-to-import-and-use-different-packages-of-the-same-name